workbench-core/datasets/src/s3DataSetStoragePlugin.test.ts

```diff
@@ -1043,6 +1044,40 @@ describe('S3DataSetStoragePlugin', () => {
         '{"Statement":[{"Action":["kms:Encrypt","kms:Decrypt","kms:ReEncrypt*","kms:GenerateDataKey*","kms:DescribeKey"],"Effect":"Allow","Principal":{"AWS":"arn:aws:iam::123456789012:root"},"Resource":"*"},{"Action":["kms:CreateGrant","kms:ListGrant","kms:RevokeGrant"],"Condition":{"Bool":{"kms:GrantIsForAWSResource":"true"}},"Effect":"Allow","Principal":{"AWS":"arn:aws:iam::123456789012:root"},"Resource":"*"}],"Version":"2012-10-17"}'
       );
     });
+
+    it('throws EndpointAlreadyExistsError when trying to create an access point you already own', async () => {
+      const s3Mock = mockClient(S3ControlClient);
+      s3Mock.on(CreateAccessPointCommand).rejects({ name: 'AccessPointAlreadyOwnedByYou' });
+
+      await expect(
+        plugin.addExternalEndpoint({
+          name,
+          path,
+          externalEndpointName,
+          ownerAccountId: mockAwsAccountId,
+          accessLevel: 'read-only',
+          externalRoleName: externalRoleArn,
+          kmsKeyArn
+        })
+      ).rejects.toThrow(EndPointExistsError);
```

Comment:
`toThrowError`

I was also just reading suggests that say you should do 
`.toThrow(<error type).toThrow('error message')